.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       with:
         ruby-version: '2.x'
     - run: gem install awesome_bot
-    - run: awesome_bot README.md --allow-redirect --allow-dupe --allow-ssl
+    - run: awesome_bot README.md --allow 429 --allow-redirect --allow-dupe --allow-ssl
```

Comment:
It seems that `awesome_bot` accepts a ` --request-delay [seconds]` parameter. It would lengthen build time so we probably don't want to set it too high, but even 1 second might go a long way.

That being said, I reckon keeping the `--allow 429` make a good fallback.

If for whatever reason we can't set request delay, then I'm happy to accept the PR as is.